runtime/tests/mock.rs

```diff
@@ -27,30 +27,45 @@ use sp_runtime::Perbill;
 pub const GENESIS_HASH: [u8; 32] = [69u8; 32];
 pub const SPEC_VERSION: u32 = VERSION.spec_version;
 
-pub fn generate_initial_authorities() -> Vec<AuthorityKeys> {
-	vec![
-		get_authority_keys_from_seed("Alice"),
-		get_authority_keys_from_seed("Bob"),
-		get_authority_keys_from_seed("Charlie"),
-	]
+pub fn generate_initial_authorities(n: usize) -> Vec<AuthorityKeys> {
```

Comment:
Since this is a public function, we cannot ensure that n < 6. I think we should do some kind of boundary checks

Reply:
Thanks for spotting this!